src/models/tests/objectiveStatus.test.js

```diff
@@ -0,0 +1,378 @@
+        activityReportId: reportOne.id,
+        objectiveId: objective.id,
+        status: 'Completed',
+      });
+
+      ActivityReportObjective.create({
+        activityReportId: reportTwo.id,
+        objectiveId: objective.id,
+        status: 'In Progress',
+      });
+
+      ActivityReportObjective.create({
+        activityReportId: reportThree.id,
+        objectiveId: objective.id,
+        status: 'Suspended',
+      });
+
+      ActivityReportObjective.create({
+        activityReportId: reportOnlyUsingObjective.id,
+        objectiveId: objectiveTwo.id,
+        status: 'Completed',
+      });
+
+      ActivityReportObjective.create({
+        activityReportId: reportOnlyUsingObjective.id,
+        objectiveId: objectiveTwoB.id,
+        status: 'Completed',
+      });
+    } catch (e) {
+      auditLogger.error(JSON.stringify(e));
+      throw e;
+    }
+  });
+  afterAll(async () => {
+    await ActivityRecipient.destroy({
+      where: {
+        activityReportId: [
+          reportOne.id,
+          reportTwo.id,
+          reportThree.id,
+          reportOnlyUsingObjective.id,
+        ],
+      },
+    });
+    await ActivityReportObjective.destroy({
+      where:
+      {
+        activityReportId: [
+          reportOne.id,
+          reportTwo.id,
+          reportThree.id,
+          reportOnlyUsingObjective.id],
+      },
+    });
+    await ActivityReportGoal.destroy({
+      where:
+      {
+        activityReportId: [
+          reportOne.id,
+          reportTwo.id,
+          reportThree.id,
+          reportOnlyUsingObjective.id,
+        ],
+      },
+    });
+    await ActivityReport.destroy({
+      where: {
+        id: [
+          reportOne.id,
+          reportTwo.id,
+          reportThree.id,
+          reportOnlyUsingObjective.id],
+      },
+    });
+    await Objective.destroy({ where: { id: [objective.id, objectiveTwo.id, objectiveTwoB.id] } });
+    await Goal.destroy({ where: { id: [goal.id, goalTwo.id] } });
+    await Grant.destroy({ where: { id: [grantOne.id, grantTwo.id, grantThree.id, grantFour.id] } });
+    await Recipient.destroy({
+      where:
+      { id: [recipientOne.id, recipientTwo.id, recipientThree.id, recipientFour.id] },
+    });
+    await User.destroy({ where: { id: user.id } });
+    await db.sequelize.close();
+  });
+  it('correct objective status moving to approved and from approved', async () => {
+    // Get report to approve.
+    const preReport = await ActivityReport.findOne(
+      { where: { id: reportOne.id }, individualHooks: true },
+    );
+    // Approve report.
+    await preReport.update(
+      { calculatedStatus: REPORT_STATUSES.APPROVED, submissionStatus: REPORT_STATUSES.SUBMITTED },
+    );
+    // Assert correct status.
+    let objectivesUpdated = await Objective.findAll({
+      where: { id: objective.id },
+    });
+    expect(objectivesUpdated.length).toBe(1);
+    expect(objectivesUpdated[0].status).toBe('Completed');
```

Comment:
```suggestion
    expect(objectivesUpdated[0].status).toBe('Complete');
```

Reply:
I think you are confusing PR's. 

This change was done in: https://github.com/HHS/Head-Start-TTADP/pull/1048

Reply:
Ah Ic it got merged. 🙃